source/bundle-dependencies.js

```diff
@@ -25,5 +25,5 @@ export default (bundler, file) => {
 			return bundleFile(bundler, dependency);
 		});
 
-	return bundled[bundled.length - 1];
+	return bundled[bundled.length - 1] || bundler;
```

Comment:
Make this `return bundler`, should be equivalent https://github.com/sinnerschrader/patternplate-transform-browserify/pull/1/commits/a2cf274b3aa4d4413112353dcc7811df6d89866d#diff-1b8746c8e0f3a22d35aa8515833715d3L23 could become `forEach` then line 25 does not need the return